### ISSUE TITLE
Docs publishing in CI is failing

### ISSUE THREAD

**Issue by bpringe** [BetterThanTomorrow/calva#1909]
```sh
ERROR    -  Required dependencies of "social" plugin not found. Install with: pip install pillow cairosvg
```

https://app.circleci.com/pipelines/github/BetterThanTomorrow/calva/5649/workflows/6112fd4d-bf5f-48e8-be23-6c7c92e91a28/jobs/25891

**Comment by PEZ** [comment]
Thanks! I fixed this with 549687241a3718eb40d7f16da1411106ee72be4a

No idea why we all of sudden need to explicitly install cairosvg and pillow, but guessing mkdocs-insiders must have changed (that requirement isn't locked down to any specific version).

Note that we don't use the requirements file in CI. There we just manually `pip install` the things. We probably should use the requirements file, because confusing.

**Comment by bpringe** [comment]
Glad it's fixed!

> We probably should use the requirements file, because confusing.

I agree.
